### PR TITLE
Automatically install nvidia device plugin for GPU instances

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [x] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: pkg/eks/tasks.go [hunk 1]
```diff
@@ -212,22 +239,26 @@ func (c *ClusterProvider) CreateExtraClusterConfigTasks(cfg *api.ClusterConfig,
...
 	tasks := &manager.TaskTree{
 		Parallel:  false,
-		IsSubTask: true,
+		IsSubTask: false,
 	}
 	var reallyInstallNeuronDevicePlugin bool
 	for _, ng := range cfg.NodeGroups {
 		reallyInstallNeuronDevicePlugin = reallyInstallNeuronDevicePlugin || api.HasInstanceType(ng, utils.IsInferentiaInstanceType)
 	}
+	var reallyInstallNvidiaDevicePlugin bool
```
Comment (aclevername) [inline pkg/eks/tasks.go:251]: really? 😂

Comment (aclevername) [inline pkg/eks/tasks.go:251]: could we rename this? having `reallyInstallNeuronDevicePlugin` and `installNeuronDevicePluginParam` is confusing

**Review by aclevername — APPROVED** [review]
lgtm

File: pkg/eks/tasks.go (hunk 1 above)
Comment (michaelbeaumont) [reply pkg/eks/tasks.go]: done

**Pull request merged**